OsmAnd/src/net/osmand/plus/mapcontextmenu/builders/AmenityMenuBuilder.java

```diff
@@ -550,16 +560,16 @@ public void buildInternal(View view) {
 			AmenityInfoRow row;
 			if (isDescription) {
 				row = new AmenityInfoRow(key, R.drawable.ic_action_note_dark, textPrefix,
-						vl, collapsable, collapsableView, 0, false, true,
-						true, 0, "", false, false, matchWidthDivider, 0);
+						vl, null, collapsable, collapsableView, 0, false,
+						true, true, 0, "", false, false, matchWidthDivider, 0);
 			} else if (icon != null) {
-				row = new AmenityInfoRow(key, icon, textPrefix, vl, collapsable, collapsableView,
-						textColor, isWiki, isText, needLinks, poiTypeOrder, poiTypeKeyName,
-						isPhoneNumber, isUrl, matchWidthDivider, 0);
+				row = new AmenityInfoRow(key, icon, textPrefix, vl, socialMediaUrl, collapsable,
```

Comment:
is it different code style applied comparing to other team members?

Reply:
Do you mean I've changed position (not order) of constructor arguments?